.github/workflows/nightly.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '8'
+          java-version: '17'
```

Comment:
For the nightly and compilation version, we need to stick to Java SE 11. That's the version Jakarta EE 10 mandates. Java SE 17 is optional.

Reply:
change to Java 11